### ISSUE TITLE
[BITV] 9.4.1.2/3.8 - In the "Details" section under the "Activity" tab, it is possible to open a dropdown list when entering "@", ":" or "/" into the textbox, for example, to select from a list of emojis. (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41896]
The elements in these lists are not accessible to screen reader users. 

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n8

9.4.1.2/11.8 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v11-n8

**Comment by susnux** [comment]
Needs to be fixed with `@nextcloud/vue` NcRich...

**Comment by emoral435** [comment]
This issue also finished #41879

**Comment by JuliaKirschenheuter** [comment]
no "pending dependency update" anymore. Closing issue

**Comment by szaimen** [comment]
Seems to work but the smart picker seems to be broken 😢

https://github.com/nextcloud/server/assets/42591237/3ca2c572-2bbb-4fdd-9623-efa367ef2ff5

**Comment by JuliaKirschenheuter** [comment]
> Seems to work but the smart picker seems to be broken 😢

https://github.com/nextcloud/server/issues/43477

**Comment by szaimen** [comment]
Actually it is https://github.com/nextcloud/server/issues/43478
